### ISSUE TITLE
Reproducing 4k/8k segmentation results

### ISSUE THREAD

**Issue by manuelknott** [wl-zhao/VPD#30]
Hi,
Can you share the config you used to train 4k and 8k iters? Most importantly, the learning rate schedule you were using?

Trying to reproduce the results.

Thank you.

**Comment by nkondapa** [comment]
I have the same question

**Comment by manuelknott** [comment]
Using the 80k schedule that you provided in the repo and just reducing the total number of epochs gives ~39 mIoU while you are reporting 43.9 in the paper for 4k steps. So I assume you are using a different config for 4k and 8k iters. Would be very thankful if you could indicate what is different there. Thanks in advance.

**Comment by manuelknott** [comment]
From the paper:
_For the fast schedule (8K iterations), we linear scale
the learning rate schedule and set the warming-up iterations
to 150._ 

Is it the same for 4k? And what does linear scale exactly mean?
